site/network/zen/network.yaml

```
@@ -14,8 +14,8 @@ packages:
       - 'fabrics=cma,ofi,xpmem'
       - '+internal-pmix'
       - '+cray-xpmem'
-    prefer: []
-      #- '+cuda'
+    prefer:
+      - '@5:'
```

is there any reason why this is not aligned with other clusters?

```suggestion
      - '@5.0.9:'
```